GOVERNMENT.md

```diff
@@ -0,0 +1,56 @@
+# Governance
+
+## Mission Statements
+### SNIRF
+The SNIRF specification serves to provide a binary file format for functional near-infrared spectroscopy data which is portable and widely applicable. In particular, the SNIRF specification seeks to capture:
+- Acquired raw data in various modalities, including auxiliary physiological and stimulus signals
+- Probe geometry and optode parameters
+- Simple derivative signals and post-processed data
+- Subject information and simple experimental metadata
+
+### SNIRF Government
+The mission of the SNIRF government is to ensure the format's longevity and to encourage its adoption. Critical to this mission is the ability of the fNIRS community to expand or correct the specification as new developments emerge without creating issues and conflicts for users. The SNIRF government is responsible for moderating this process, and for collecting and disseminating important information to SNIRF users.
+
+## Bylaws
+
+## Article I. Steering Committee
+
+**Section 1.** Steering Committee Definition. The SNIRF Steering Committee is ultimately responsible for guiding the development of SNIRF in accordance with its mission. The committee must unanimously approve the releases of official and final changes to the specification, as well as changes to the SNIRF government or mission statement.
+
+**Section 2.** Election. Steering committee members must be nominated or self-nominated in advance of the end of a sitting member's term. The departing member is responsible for the organization of the election, and cannot be elected to remain in the same seat. All SNIRF Contributors, Maintainers, and Steering Committee members are elligble to vote in the election.
+
+**Section 3.** Terms. The Steering Committee is composed of between 3 and 5 members of the fNIRS community who are elected to serve 3 year terms.
+
+**Section 4.** Meetings. A majority of Steering Committee members must meet at least biannually. At least one Maintainer must be present at these meetings. These meetings should review the following:
+- The most recent Maintainers Report
+- Any pending releases or critical changes
+- project priorities and progress towards goals
+- Delegation of resources (i.e maintainers' attention) towards completion of goals
+
+The meeting should result in the authorship of a brief report.
+
+**Section 5.** Steering Committee Reports. The Steering Committee must author a brief report corresponding following each of its required biannaual meetings. This report must document the actions and considerations of the committee, and must be archived and disseminated via the GitHub repository. The report must include the names of the Steering Committee's members.
```

Comment:
I think we have to delete "corresponding" from the first sentence so that it reads "...brief report following each of ..."